recipe/meta.yaml

```diff
@@ -26,7 +26,7 @@ requirements:
 
   run:
     - python
-    - argon2_cffi >=16.2
+    - argon2-cffi >=19.2.0
```

Comment:
Is this a global name change?

Reply:
yes, `argon2<=19.1.X` is shipped by `argon2_cffi` and `argon2>=19.2.0` is shipped by `argon2-cffi`